src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
+    type--footer();
+    align-items: center;
     background: $color--greyscale--white;
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+    
+    // Remove from the document flow and hide. Can't use display: none due to
+    // https://github.com/tj/nib/issues/291
+    position: absolute;
+    visibility: hidden;
+    +mq--show-footer() {
+        position: relative;
+        visibility: visible;
+    }
+
```

Comment:
gonna need some whitespace guidelines. currently, we don't try to throw in whitespaces between nested rules unless to separate parent rules vs descendant rules.